src/backend/persons/main.go

```diff
@@ -60,13 +60,13 @@ func getPersons(table dynamo.Table) events.APIGatewayProxyResponse {
 			fmt.Sprintf("scan error: %s", err.Error()))
 	}
 
-	jsonBytes, err := json.Marshal(persons)
+	json, err := json.Marshal(persons)
```

Comment:
Ref: https://dave.cheney.net/2019/01/29/you-shouldnt-name-your-variables-after-their-types-for-the-same-reason-you-wouldnt-name-your-pets-dog-or-cat